### DISCUSSION TITLE
Working directory

### DISCUSSION THREAD

**Discussion by treizexiii** [Jeffser/Alpaca#371] [Ideas]
Hi!
Your app is very nice. 
I was wondering if it's possible to discuss with a model by providing it with a folder to base itself on. I particularly thought of a folder containing source code.

**Comment by Jeffser** [comment]
Hi, thanks for the suggestion, whilst that is possible I think it could overload most models, of course that depends on each user's computer but giving a model all the text from potentially dozens of files on a single query might not be a good idea
